plugins/auth-backend-module-oidc-provider/src/authenticator.ts

```diff
@@ -153,7 +153,8 @@ export const oidcAuthenticator = createOAuthAuthenticator({
         .start(input, {
           ...options,
         })
-        .then(resolve);
+        .then(resolve)
+        .catch(reject);
```

Comment:
Can we remove the `strategy.error = reject` above now that this has been added? That most likely breaks in horrible ways anyway. More generally it should now be enough to replace this entire `new Promise` block with just `return helper.start(...)`